### PR TITLE
Made more changes

### PR DESCRIPTION
**Pull request by CBID2**
Hi Amanda! :) Here are some other changes I made to the ReadMe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ This is a living document that can serve as a resource for finding people includ
 
 This resource began in reponse to a discussion post started by [Christine Belzie](https://github.com/CBID2) titled [Where are all the women owned Open Source projects](https://dev.to/cbid2/where-are-all-the-women-owned-open-source-projects-4pd0). In looking for some resources to send Christine, I was unable to find anything thorough or currently maintained and decided to start this project.  It is our hope that this resource will get so large that we will have to find a better solution to hold all the amazing resources.
 
-Contributions to this resource are welcome and encouraged. 
+Contributions to this resource are welcome. ***NOTE**: To ensure that this repository stays true to its mission, we will prioritize open source projects and related content that is created by womxn,especially those from other marginalized communities.
```
Comment (amandamartin-dev) [inline README.md]: I don't think this is necessary, please remove and we can revisit the readme in the future if there seems to be confusion on the purpose of if.

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CBID2) [reply README.md]: Ok, I just think it’d be beneficial to have that disclaimer to ensure that our targeted audience is represented. In my experience being in women-centric coding spaces, women of color are often overlooked, so I want to ensure that they are amplified.

Comment (amandamartin-dev) [reply README.md]: It's already implied by the existence of the repo adn the rest of the description.  The intro is already quite verbose and there have been no indications there is confusion from submitters yet.  This issue can be revisited should confusion arise from contributors

Comment (CBID2) [reply README.md]: Ah okay.

Comment (amandamartin-dev) [reply README.md]: Yeah, totally understand your point but haven't seen confusion about the repo yet so I don't want to add bulk unless folks are coming to us with confusion. Appreciate the discussion always

Comment (CBID2) [reply README.md]: You're welcome! I just noticed that people tend to not mention womxn from other marginalized groups unless that disclaimer is there.

Comment (amandamartin-dev) [reply README.md]: it's in the first sentence of the readme so I think it's clear

Comment (CBID2) [reply README.md]: Oh no I was speaking for other women-centric initiatives and ranting a little lol. We're good 👍🏾

File: README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ This is a living document that can serve as a resource for finding people includ
 
 This resource began in reponse to a discussion post started by [Christine Belzie](https://github.com/CBID2) titled [Where are all the women owned Open Source projects](https://dev.to/cbid2/where-are-all-the-women-owned-open-source-projects-4pd0). In looking for some resources to send Christine, I was unable to find anything thorough or currently maintained and decided to start this project.  It is our hope that this resource will get so large that we will have to find a better solution to hold all the amazing resources.
 
-Contributions to this resource are welcome and encouraged. 
+Contributions to this resource are welcome and encouraged.
 
-When adding someone please add their name, link to their github profile, project name, and link to project.
+Click on the [CONTRIBUTING.md](https://github.com/amandamartin-dev/women-led-open-source/blob/main/CONTRIBUTING.md) to learn how to submit resources.
```
Comment (CBID2) [inline README.md:9]: Is this one ok @amandamartin-dev? I thought linking the contributing file would make it easier for readers to learn how to contribute.

Comment (amandamartin-dev) [reply README.md:9]: oh yes! 

also, just had another thought about the other section - I think that could be useful in the contributing.md as we didn't add the context there to remind folks what types of ppl we want to highlight in this resource. wdyt?

**Review by amandamartin-dev — APPROVED** [review]
lgtm

**Comment by CBID2** [comment]
> lgtm

Alright! 😊

File: README.md (hunk 2 above)
Comment (CBID2) [reply README.md:9]: I like that! :)

Comment (CBID2) [reply README.md:9]: > oh yes!
> 
> also, just had another thought about the other section - I think that could be useful in the contributing.md as we didn't add the context there to remind folks what types of ppl we want to highlight in this resource. wdyt?

Made an issue for it: https://github.com/amandamartin-dev/women-led-open-source/issues/16

**Pull request merged**